main/EDMainMod.F90

```diff
@@ -15,6 +15,7 @@ module EDMainMod
   use FatesInterfaceMod        , only : hlm_current_day 
   use FatesInterfaceMod        , only : hlm_use_planthydro 
   use FatesInterfaceMod        , only : hlm_reference_date
+  use FatesInterfaceMod        , only : hlm_use_ed_prescribed_phys
```

Comment:
Should we transition eventually to use_fates instead of use_ed for the switches?

Reply:
I don't have any strong feelings on this.  I originally thought of dynamics as the "ed part of fates", so I left the ed in "hlm_use_ed_st3", since it is referring to turning off dynamics.  I'm fine with changing the names.
If we do, now would be a good time to do it, because this PR will pair with a change on the HLM side...